src/parser.mjs

```diff
@@ -177,7 +178,18 @@ const createParser = () => {
   const parseApiDocs = async apiDocs => {
     // We do a Promise.all, to ensure that each API doc is resolved asynchronously
     // but all need to be resolved first before we return the result to the caller
-    const resolvedApiDocEntries = await Promise.all(apiDocs.map(parseApiDoc));
+
+    const progressBar = createProgressBar('Parsing API Docs');
+    progressBar.start(apiDocs.length, 0);
+
+    const resolvedApiDocEntries = await Promise.all(
+      apiDocs.map(async apiDoc => {
+        progressBar.increment();
+        return await parseApiDoc(apiDoc);
```

Comment:
No need of adding `return await` the function itself is already async, so you can drop the `async apiDoc` and simply do `return parseApiDoc`

Since the map function doesn't do any async work itself.